tmail-backend/deployment-tests/distributed-es6-backport/pom.xml

```diff
@@ -11,11 +11,6 @@
 
     <artifactId>deployment-tests-distributed-es6-backport</artifactId>
 
-    <properties>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
-    </properties>
```

Comment:
Why don't you switch to 17 but remove this? My guess is that your default JDK is 17 now, but I think it could be different for everyone else/ other env.

Reply:
this property is unnecessary, 
we already have a maven configuration compiler in the parent pom. 
I guess this line has been auto-generated when we create a new module and forget to remove this.